package.json

```diff
@@ -16,6 +16,7 @@
         "build:performance": "yarn build:performance:components && yarn build:performance:benchmarks",
         "build:performance:components": "nx build @lwc/perf-benchmarks-components",
         "build:performance:benchmarks": "nx build @lwc/perf-benchmarks",
+        "check-license-headers": "node ./scripts/tasks/check-license-headers.js",
```

Comment:
But that file is being deleted in this PR?

Reply:
Oops, missed a spot. Thanks for catching it!